tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -99,17 +94,22 @@ spec:
           repository=$(jq -r '.repository' <<< $component)
           imageTags=$(jq '.tags' <<< $component)
 
+          AUTH_FILE=$(mktemp)
+          select-oci-auth "${containerImage}" > "$AUTH_FILE"
+
           # Just read the first from the list of architectures
           read -r arch_json <<< $(get-image-architectures "${containerImage}")
+          os=$(echo "${arch_json}" | jq -r .platform.os)
           arch=$(echo "${arch_json}" | jq -r .platform.architecture)
           name=$(jq -r '.name' <<< $component)
           git_sha=$(jq -r '.source.git.revision' <<< $component) # this sets the value to "null" if it doesn't exist
+          media_type=$(skopeo inspect --raw "docker://${containerImage}" | jq -r .mediaType)
+          oras_args=()
+          if [ "$media_type" == "application/vnd.docker.distribution.manifest.list.v2+json" ]; then
```

Comment:
This would also apply for media type `"application/vnd.oci.image.index.v1+json"`, no?

Another option would be to check if the digest in the `containerImage` pull spec equals the digest in `arch_json`. If they're different, it means it's a multi arch image.